src/assets/i18n/de.json

```diff
@@ -96,6 +97,35 @@
     "PARTICIPATION": "Teilnahme",
     "PREDICTION_SET": "Vorhersage für Terminteilnahme erfolgreich gespeichert"
   },
+  "editappointments": {
+    "CREATE": "Termin erstellen",
+    "EDIT": "Termin bearbeiten",
+    "SAVECLOSE": "Speichern und schließen",
+    "SAVECONTINUE": "Speichern und weiter",
+    "BASICDATA": "Basisdaten",
+    "ADDITIONALDATA": "Details",
+    "PARTICIPATION": "Teilnahmestatus",
+    "TITLE": "Terminbezeichnung *",
```

Comment:
Wie auch schon weiter oben beim Status. Es ist unglücklich wenn der State also das Sternchen Teil der Übersetzung ist. Bitte überall das Sternchen raus und direkt im Label hinzufügen.

`<label for="title">{{ 'editappointments.TITLE' | translate }} *</label>`